### DISCUSSION TITLE
TempLogin 컴포넌트 정말필요한가?!

### DISCUSSION THREAD

**Discussion by NamgyungKim** [prgrms-fe-devcourse/FEDC2_2JAVATAYO_Dongyoung#160] [Ideas]
PR: #157 

- 로그인과 로그아웃했을경우를 비교하며 작업해야하는경우,
- 다른사람으로 로그인할때 email, pw를 치는게 귀찮다. (지금으로서는 로컬스토리지의 토큰 변경)
- 여러 상황드들을 테스트할 경우, ex)로그인안한 상태로 create, update 페이지를 가거나, 로그인상태로 회원가입페이지에 갔을 경우 등...

- [x] 필요하다. 🙆‍♀️
- [ ] 필요없다. 🙅‍♂️

**Comment by YuHyun-P** [comment]
로그인 API를 쏘는 것보다 토큰 자체를 상수로 만들어서 만든 버튼을 누를 때마다 로컬 스토리지 토큰 값이 변경되는 방법도 있겠네요!

**Comment by Hyevvy** [comment]
오 좋네용!! 테스트할 때 좋을 거 같습니다👍
사실 저는 유저 변경에 대해서는 쓰지는 않지만 굿 아이디어인 것 같습니당 😳
